src/ApparentHorizons/StrahlkorperGr.cpp

```diff
@@ -183,6 +185,55 @@ Scalar<DataVector> area_element(
            square(get(dot_product(cap_theta, cap_phi, spatial_metric))));
   return area_element;
 }
+
+template <typename Frame>
+Scalar<DataVector> spin_function(
+    const StrahlkorperTags::StrahlkorperTags_detail::Jacobian<Frame>& tangents,
+    const YlmSpherepack& ylm,
+    const tnsr::I<DataVector, 3, Frame>& unit_normal_vector,
+    const Scalar<DataVector>& area_element,
+    const tnsr::ii<DataVector, 3, Frame>& extrinsic_curvature) noexcept {
+  auto sin_theta = make_with_value<Scalar<DataVector>>(area_element, 0.0);
+  get(sin_theta) = sin(ylm.theta_phi_points()[0]);
+
+  auto extrinsic_curvature_theta_normal_sin_theta =
+      make_with_value<Scalar<DataVector>>(area_element, 0.0);
+  auto extrinsic_curvature_phi_normal =
+      make_with_value<Scalar<DataVector>>(area_element, 0.0);
+  for (size_t i = 0; i < 3; ++i) {
+    DataVector extrinsic_curvature_dot_normal(get(area_element).size(), 0.0);
```

Comment:
Use the sin_theta allocation here

Reply:
Do you mean sin_theta instead of area_element, or initialize to sin_theta?